### ISSUE TITLE
pdf manual is incomplete

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#4925]
- [ ] some .out files are not generated, possibly because of https://github.com/geodynamics/aspect/commit/042a06f70c8def5dda32478a04dda1824fc9c00b#diff-f48ddfd1eec38c4d39f84e195259371f8397cd30146f46b113bdcb4590ed262c (I don't understand this change). It looks like we only make cookbooks/*/doc/*prm but no longer cookbooks/*prm:
```
! Package Listings Error: File `convection-box.prm(.out)' not found.
```
- [ ] None of the cookbooks/benchmarks works because of https://github.com/geodynamics/aspect/commit/9ae0e7b9102356fdf34cda1efda1fb8da22c4c5a
- [ ] Images like visit0000.png are not found

**Comment by sac-bsa** [comment]
As you note 042a06f70c8def5dda32478a04dda1824fc9c00b limits what *.prm.out files are generated. A part of that PR (and the many other cookbook documentation moving PRs) was ensuring that there were no `cookbooks/*.prm` or `cookbooks/*/*.prm` files that are meant to be included in the documentation. Hence `convection-box.prm.out` is generated from `cookbooks/convection-box/doc/convection-box.prm` in the same directory. This change solved a problem that the original rule had with trying to convert files that were not in doc directories, and place them into doc directories. As I moved documentation, this affects several benchmarks, such as `benchmarks/solkz/compositional_fields/solkz_particles.prm` which the original rule attempted to write into `benchmarks/solkz/compositional_fields/docs/solkz_particles.prm.out`. Several cookbooks and benchmarks had relied on this behavior prior to being relocated, but I standardized it as I went. 

All that being said, 9ae0e7b9102356fdf34cda1efda1fb8da22c4c5a seems to be related to all three of the noted problems. The newclude package doesn't appear to actually allow relative paths, which would cause images and .out files to not be found. I tried reverting it, and it appears that I can now successfully build the manual (including cookbooks, listings of .prm.out files, and images), although as #4780 would expect, doing so breaks the CMake manual building.
